pkg/server/sandbox_run.go

```diff
@@ -241,10 +232,9 @@ func (c *criContainerdService) RunPodSandbox(ctx context.Context, r *runtime.Run
 	}
 
 	// Add sandbox into sandbox store.
-	meta.CreatedAt = time.Now().UnixNano()
-	if err := c.sandboxStore.Create(meta); err != nil {
-		return nil, fmt.Errorf("failed to add sandbox metadata %+v into store: %v",
-			meta, err)
+	sandbox.CreatedAt = time.Now().UnixNano()
```

Comment:
We don't maintain state of sandbox. We infer that directly from sandbox container state.

We could not do that for container because we also need exit code, finished timestamp etc. which we have to checkpoint ourselves. But for sandbox, we only care about whether it is running or not.